### PR TITLE
enable wrapper divs

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

File: README.md [hunk 1]
```diff
@@ -97,21 +97,37 @@ This works for elements in the header as well:
...
 
 ```html
 <!-- A comment -->
 <div>
   <!-- This does not work -->
   <amp-img src="img/image1.jpg" width="200" height="100" layout="responsive"></amp-img>
 </div>
+<div>
+  <!-- This does work though -->
+  <amp-img src="img/image1.jpg" width="200" height="100" layout="responsive"></amp-img>
```
Comment (kul3r4) [inline README.md]: what's the difference from the tag above?

Comment (sebastianbenz) [inline README.md]: good point - updated the doc

**Comment by sebastianbenz** [comment]
PTAL

**Pull request merged**